spec/index.html

```diff
@@ -1381,12 +1381,35 @@ <h2>Fragment Identifiers</h2>
 </section>
 
 <section id="section-generalized-rdf" class="informative">
-  <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
+  <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
 
   <p>It is sometimes convenient to loosen the requirements
     on <a>RDF triples</a>.  For example, the completeness
-    of the RDFS entailment rules is easier to show with a
-    generalization of RDF triples.</p>
+    of the RDFS entailment rules is easier to show with
+    symmetric RDF triples.</p>
```

Comment:
```suggestion
    a symmetric notion of RDF triples.</p>
```

Reply:
I'm not sure what's meant by the above. Perhaps this would serve?
```suggestion
    a notion of symmetric RDF triples.</p>
```

Reply:
Oops. Of course. I got the order of the words wrong.